src/main/java/ch/njol/skript/expressions/ExprPlayerViewDistance.java

```diff
@@ -25,42 +25,33 @@
 import ch.njol.skript.doc.Name;
 import ch.njol.skript.doc.RequiredPlugins;
 import ch.njol.skript.doc.Since;
-import ch.njol.skript.expressions.base.PropertyExpression;
-import ch.njol.skript.lang.Expression;
-import ch.njol.skript.lang.SkriptParser;
-import ch.njol.util.Kleenean;
+import ch.njol.skript.expressions.base.SimplePropertyExpression;
 import ch.njol.util.coll.CollectionUtils;
+import org.apache.commons.lang.NotImplementedException;
 import org.bukkit.Bukkit;
 import org.bukkit.entity.Player;
 import org.bukkit.event.Event;
 import org.eclipse.jdt.annotation.Nullable;
 
 @Name("View Distance")
-@Description("The view distance of a player. Can be changed.")
+@Description({"The view distance of a player. Can be changed.",
+		"NOTE: This may not work on some versions (such as MC 1.14.x).",
+		"The return value in this case will be the view distance set in system.properties."})
```

Comment:
```suggestion
@Description({
		"The view distance of a player. Can be changed.",
		"NOTE: This may not work on some versions (such as MC 1.14.x).",
		"The return value in this case will be the view distance set in system.properties."})
```

Reply:
I'll fix the indentation, but I don't understand why you're asking me to push it to the next line?